From: Nithin Sujir <nsujir@broadcom.com>
Date: Fri, 6 Dec 2013 09:53:16 -0800
Subject: [PATCH 081/108] tg3: Don't add rxbds_empty to rx_over_errors
Patch-mainline: v3.14-rc1
Git-commit: ec99f76d281bae0194c6060417d3465ded94f0ed
References: bsc#908458 FATE#317507


rxbds_empty is an informational statistic signifying that a ring full
condition was observed. It does not mean an overflow has occurred.

Signed-off-by: Nithin Nayak Sujir <nsujir@broadcom.com>
Signed-off-by: Michael Chan <mchan@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/broadcom/tg3.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ethernet/broadcom/tg3.c
+++ b/drivers/net/ethernet/broadcom/tg3.c
@@ -11719,8 +11719,6 @@ static void tg3_get_nstats(struct tg3 *t
 		get_stat64(&hw_stats->rx_frame_too_long_errors) +
 		get_stat64(&hw_stats->rx_undersize_packets);
 
-	stats->rx_over_errors = old_stats->rx_over_errors +
-		get_stat64(&hw_stats->rxbds_empty);
 	stats->rx_frame_errors = old_stats->rx_frame_errors +
 		get_stat64(&hw_stats->rx_align_errors);
 	stats->tx_aborted_errors = old_stats->tx_aborted_errors +
